pip/download.py

```diff
@@ -369,12 +371,28 @@ def __init__(self, *args, **kwargs):
         for host in insecure_hosts:
             self.mount("https://{0}/".format(host), insecure_adapter)
 
-    def request(self, method, url, *args, **kwargs):
+    def request(self, *args, **kwargs):
         # Allow setting a default timeout on a session
         kwargs.setdefault("timeout", self.timeout)
 
+        # If we're running inside of a zipimport, and we were not given an
+        # explicit certificate, then we want to unzip our bundled certificate.
+        if (not isinstance(self.verify, six.string_types) and
+                isinstance(
+                    getattr(requests, "__loader__", None),
+                    zipimport.zipimporter)):
+            cert_data = pkgutil.get_data("pip._vendor.requests", "cacert.pem")
+            if cert_data is not None:
+                with tempfile.NamedTemporaryFile(delete=False) as temp_cert:
```

Comment:
Related Python issue: [tempfile.NamedTemporaryFile not particularly useful on	Windows](http://bugs.python.org/issue14243)